holoviews/plotting/widgets/widgets.js

```diff
@@ -539,3 +539,64 @@ for (var k in _namespace) {
     window.HoloViews[k] = _namespace[k];
   }
 }
+
+var JS_MIME_TYPE = 'application/javascript';
+var HTML_MIME_TYPE = 'text/html';
+var EXEC_MIME_TYPE = 'application/vnd.holoviews_exec.v0+json';
+var CLASS_NAME = 'output';
+
+/**
+ * Render data to the DOM node
+ */
+function render(props, node) {
+  var script = document.createElement("script");
+  node.appendChild(script);
+}
+
+/**
+ * Handle when a new output is added
+ */
+function handleAddOutput(event, handle) {
+  var output_area = handle.output_area;
+  var output = handle.output;
+  // limit handleAddOutput to display_data with EXEC_MIME_TYPE content only
+  if ((output.output_type != "display_data") || (!output.data.hasOwnProperty(EXEC_MIME_TYPE))) {
+    return
+  }
+  var toinsert = output_area.element.find("." + CLASS_NAME.split(' ')[0]);
+  if (output.metadata[EXEC_MIME_TYPE]["id"] !== undefined) {
+    toinsert[0].firstChild.textContent = output.data[JS_MIME_TYPE];
+    output_area._hv_plot_id = output.metadata[EXEC_MIME_TYPE]["id"];
+  }
+}
+
+function register_renderer(events, OutputArea) {
+  function append_mime(data, metadata, element) {
+    // create a DOM node to render to
+    var toinsert = this.create_output_subarea(
+    metadata,
+    CLASS_NAME,
+    EXEC_MIME_TYPE
+    );
+    this.keyboard_manager.register_events(toinsert);
+    // Render to node
+    var props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};
+    render(props, toinsert[0]);
+    element.append(toinsert);
+    return toinsert
+  }
+  events.on('output_added.OutputArea', handleAddOutput);
+
+  OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {
+    safe: true,
+    index: 0
+  });
+}
+
+if (window.Jupyter !== undefined) {
```

Comment:
@jlstevens Any way of stopping this from running in labmode? IIRC you do define ``window.Jupyter``.

Reply:
Added the try/catch you suggested.